alpakaConfig.cmake

```diff
@@ -488,8 +490,13 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     LIST(APPEND CUDA_NVCC_FLAGS "-Wno-deprecated-gpu-targets")
                 ENDIF()
 
-                LIST(APPEND CUDA_NVCC_FLAGS "--expt-extended-lambda")
-                LIST(APPEND CUDA_NVCC_FLAGS "--expt-relaxed-constexpr")
+
+                IF(ALPAKA_CUDA_NVCC_LAMBDAS)
+                    LIST(APPEND CUDA_NVCC_FLAGS "--expt-extended-lambda")
+                ENDIF()
+                IF(ALPAKA_CUDA_NVCC_CONSTEXPR)
```

Comment:
Could you use a name that is more closely related to the flag? `ALPAKA_CUDA_NVCC_RELACED_CONSTEXPR`?

Reply:
Sure, can do in both cases.